### ISSUE TITLE
Safe set is wrong using -mode EF

### ISSUE THREAD

**Issue by petergjoel** [imitator-model-checker/imitator#36]
In the attached model (see here: [Sched2.50.0.zip](https://github.com/imitator-model-checker/imitator/files/4943954/Sched2.50.0.zip)) the safe set is reported as
```
  b >= 10
 & C3_WORST >= 20
 & 50 > b + C3_WORST
 & 22 >= b
```
which includes the point `C3_WORST=19 & b=23`.

However, modifying the model to add these as initial constraints and re-running with `-mode EF` leads to the `False` predicate over the parameters (modified model included in zip).

This is also validated by translating the model to uppaal.

A possibly valid set of (negative) constraints is given in SMT-format here: 
https://rise4fun.com/Z3/xhg6
